Plugins/Statistics/GitImpact/ImpactControl.cs

```diff
@@ -545,5 +535,35 @@ public ImpactLoader.DataPoint GetAuthorInfo(string author)
                 return new ImpactLoader.DataPoint(0, 0, 0);
             }
         }
+
+        public new void Dispose()
```

Comment:
Why not just override the base?

Reply:
That's what I tried first and got the error: https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0506?f1url=%3FappId%3Droslyn%26k%3Dk(CS0506)

Reply:
Because the virtual method is `Dispose(bool)`

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.containercontrol.dispose?view=windowsdesktop-8.0#system-windows-forms-containercontrol-dispose(system-boolean)

Reply:
It was not possible either but thinking about it with a clear view, now that's OK, I got it done well...